source/learning-wazuh/detect-fs-changes.rst

```diff
@@ -119,27 +118,48 @@ the FILE INTEGRITY tab, which would look something like this:
     The default time windows in Kibana is only "Last 15 minutes" which may be too small to encompass your activities in this lab.  Click on
     the time window value and change it to something broader if needed.
 
-Where FIM data is stored
-------------------------
+A look under the hood of syscheck
+---------------------------------
 
-Each time a Wazuh agent runs a periodic syscheck FIM scan, the monitored file attributes are sent back to the Wazuh
-Manager where they are stored in an agent-specific flat file so that the results of the next scan can be compared to
-the results of the previous scan in order to detect changes.  Change history is also stored in the same flat flat.
-A real-time monitoring event on an agent conveys the same kind of information about an individual changed file back
-to the Wazuh Manager.
+What has actually happened in the background? How does Wazuh track file state between scans so it can know when a file has changed and what about that file changed?
 
-On wazuh-manager, use the text viewer of your choice to peruse the FIM history file for windows-agent.  The file
-will be called "/var/ossec/queue/syscheck/(windows-agent) any->syscheck", and if you look at the end of the file
-you should see records relevant to the c:\\apple and c:\\orange lab activities you just went through, looking like
-this:
+On the manager in the /var/ossec/queue/db/ directory we see files like 000.db, 001.db, 002.db, 003.db.  These are SQLite files for the manager and each agent (by ID#), each containing multiple tables related to a system including one related to syscheck.
```

Comment:
Folder names, file names and reserved names (like `fim_entry`) should be formatted like this:
``/var/ossec/queue/db/``
``000.db``